FreeRTOS/FreeRTOS-Plus-TCP
# Dns decouple

Decouple and refactor DNS

Description
-----------
Refactor DNS to be unit testable
add unit test

Test Steps
-----------
Unit Test passes

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
